src/routes/write-exercise/components/InteractionSection.tsx

```diff
@@ -39,7 +39,6 @@ const InteractionSection = (props: InteractionSectionProps): ReactElement => {
   const [isArticleMissing, setIsArticleMissing] = useState<boolean>(false)
   const [input, setInput] = useState<string>('')
   const [submittedInput, setSubmittedInput] = useState<string | null>(null)
-
```

Comment:
In general its best to avoid changes to formatting/code style if there is not a specific reason for it. Otherwise we will just change from one to another style because everybody likes something different.